main/src/page-content/content-script.js

```diff
@@ -138,6 +138,7 @@ function replaceCurrent(resultText) {
 }
 
 function replaceAll(rawReplaceText) {
+  console.log("RRT", rawReplaceText)
```

Comment:
No console logs. The extension must be quiet.

Reply:
Removed. It was a leftover debug log which I seem to have missed.